docs/docs/tooling/protobuf.md

```diff
@@ -0,0 +1,135 @@
+---
+sidebar_position: 1
```

Comment:
Can we rename this file to 00-protobuf.md if we want it to be the first one? It'll be better when we add more docs.